client/src/components/admin_dashboard/FeedbackModal.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from 'react';
+import { Modal, Accordion, Icon, List } from 'semantic-ui-react'
+import { makeCall } from '../../apis';
+
+export default function FeedbackModal(props) {
+    const [publicFeedback, setPublicFeedback] = useState([])
+    const [privateFeedback, setPrivateFeedback] = useState([])
+    const [testimonial, setTestimonial] = useState([])
+    const [profile, setProfile] = useState(null)
+    const [currIndex, setCurrIndex] = useState(-1)
+
+    useEffect(() => {
+        makeCall({}, `/admin/feedback/${props.userDetails}/${props.profileId}`, 'get')
+            .then((res) => {
+                setPublicFeedback(res.public)
+                setPrivateFeedback(res.private)
+                setTestimonial(res.testimonial)
+                setProfile(res.profile)
+            });
+    }, [props.profileId])
+
+    const handleClick = (e, titleProps) => {
+        const { index } = titleProps
+        const activeIndex = currIndex
+        const newIndex = activeIndex === index ? -1 : index
+        setCurrIndex(newIndex)
+    }
+
+    const constructDisplay = (feedback) => {
+        let display = [];
+        for (const response of feedback) {
+            display.push(<List.Item>{response}</List.Item>);
+        }
+        return display;
+    }
+
+    return (
+        <Modal open={props.modalOpen} onClose={props.toggleModal} closeIcon>
```

Comment:
Really like the idea of using Accordions to view all feedback in a manageable way!